src/Scripting/Core/ScriptOptions.cs

```diff
@@ -54,6 +53,7 @@ private static ImmutableArray<MetadataReference> GetDefaultMetadataReferences()
                 "System.Reflection.Extensions",
                 "System.Reflection.Primitives",
                 "System.Runtime",
+                "System.Runtime.Extensions",
```

Comment:
Odd. I think we had the tests running on CoreCLR already. Not sure why wouldn't it suddenly not work. Or maybe the tests were added since the last time I run them on Core. Can you try to run them on Core without your change to see if they pass?

---
In reply to: [110977656](https://github.com/dotnet/roslyn/pull/18471#discussion_r110977656) [](ancestors = 110977656,110973145,110970347)